packages/world/gas-report.json

```diff
@@ -2,98 +2,92 @@
   {
     "source": "test/KeysInTableModule.t.sol",
     "name": "install keys in table module",
-    "functionCall": "world.installRootModule(keysInTableModule, abi.encode(sourceTableId))",
+    "functionCall": "world.installRootModule(keysInTableModule, abi.encode(tableId))",
     "gasUsed": 1032425
   },
   {
     "source": "test/KeysInTableModule.t.sol",
     "name": "set a record on a table with keysInTableModule installed",
-    "functionCall": "world.setRecord(namespace, sourceFile, keyTuple1, abi.encodePacked(val1))",
+    "functionCall": "world.setRecord(namespace, sourceName, keyTuple1, abi.encodePacked(val1))",
     "gasUsed": 239856
   },
   {
     "source": "test/KeysInTableModule.t.sol",
     "name": "install keys in table module",
-    "functionCall": "world.installRootModule(keysInTableModule, abi.encode(sourceTableId))",
+    "functionCall": "world.installRootModule(keysInTableModule, abi.encode(tableId))",
     "gasUsed": 1032425
   },
   {
     "source": "test/KeysInTableModule.t.sol",
     "name": "change a record on a table with keysInTableModule installed",
-    "functionCall": "world.setRecord(namespace, sourceFile, keyTuple1, abi.encodePacked(val2))",
+    "functionCall": "world.setRecord(namespace, sourceName, keyTuple1, abi.encodePacked(val2))",
     "gasUsed": 30820
   },
   {
     "source": "test/KeysInTableModule.t.sol",
     "name": "delete a record on a table with keysInTableModule installed",
-    "functionCall": "world.deleteRecord(namespace, sourceFile, keyTuple1)",
+    "functionCall": "world.deleteRecord(namespace, sourceName, keyTuple1)",
     "gasUsed": 146907
   },
   {
     "source": "test/KeysWithValueModule.t.sol",
     "name": "install keys with value module",
-    "functionCall": "world.installRootModule(keysWithValueModule, abi.encode(sourceTableId))",
-    "gasUsed": 609250
+    "functionCall": "world.installRootModule(keysWithValueModule, abi.encode(tableId))",
+    "gasUsed": 1036002
   },
   {
     "source": "test/KeysWithValueModule.t.sol",
     "name": "Get list of keys with a given value",
-    "functionCall": "bytes32[] memory keysWithValue = getKeysWithValue(world, sourceTableId, abi.encode(value1))",
-    "gasUsed": 7658
-  },
-  {
-    "source": "test/KeysWithValueModule.t.sol",
-    "name": "compute the target table selector",
-    "functionCall": "bytes32 targetTableSelector = getTargetTableSelector(MODULE_NAMESPACE, sourceTableId)",
-    "gasUsed": 2240
+    "functionCall": "bytes32[][] memory keysWithValue = getKeysWithValue(world, tableId, abi.encode(value1))",
+    "gasUsed": 8679
   },
   {
     "source": "test/KeysWithValueModule.t.sol",
     "name": "install keys with value module",
-    "functionCall": "world.installRootModule(keysWithValueModule, abi.encode(sourceTableId))",
-    "gasUsed": 609250
+    "functionCall": "world.installRootModule(keysWithValueModule, abi.encode(tableId))",
+    "gasUsed": 1036002
```

Comment:
increase gas is expected because we now do a high (but constant) amount of storage writes. The cost amortises over many records - we should set up a gas report for this